File: 2025-2/sesion-de-clase-06/SoftProg/SoftProgModelo/src/main/java/pe/edu/pucp/inf30/softprog/modelo/ventas/OrdenVenta.java
package pe.edu.pucp.inf30.softprog.modelo.ventas;

import java.util.Date;
import java.util.List;
import pe.edu.pucp.inf30.softprog.modelo.Registro;
import pe.edu.pucp.inf30.softprog.modelo.clientes.Cliente;
import pe.edu.pucp.inf30.softprog.modelo.rrhh.Empleado;

/**
 *
 * @author eric
 */
public class OrdenVenta extends Registro {
    private double total;
    private Date fecha;
    private Cliente cliente;
    private Empleado empleado;
    private List<LineaOrdenVenta> lineas;

    public List<LineaOrdenVenta> getLineas() {
        return lineas;
    }

    public void setLineas(List<LineaOrdenVenta> lineas) {
        this.lineas = lineas;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    public Cliente getCliente() {
        return cliente;
    }

    public void setCliente(Cliente cliente) {
        this.cliente = cliente;
    }

    public Empleado getEmpleado() {
        return empleado;
    }

    public void setEmpleado(Empleado empleado) {
        this.empleado = empleado;
    }
    
    
}

File: 2025-2/sesion-de-clase-14/SoftProgModelo/Ventas/OrdenVenta.cs
using System;
using System.Collections.Generic;
using PUCP.SoftProg.Modelo.Clientes;
using PUCP.SoftProg.Modelo.RRHH;

namespace PUCP.SoftProg.Modelo.Ventas {
    public class OrdenVenta : Registro {
        public Cliente Cliente { get; set; }
        public Empleado Empleado { get; set; }
        public List<LineaOrdenVenta> LineasOrdenVenta { get; set; }
        public double Total {  get; set; }
        public DateTime FechaHora { get; set; }
    }
}